content/en/docs/bootstrap/airgapped.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Bootstrap Flux in an Air-gapped Environment"
```

Comment:
Can we use `title` and `linkTitle`? The titles in the sidebar get a little long?

Reply:
Potentially also use `weight`, Air-gapped is probably not the most searched for article?

Reply:
We should preserve the order from the original guide, Git/GitHub/GitLab/etc.

Reply:
Agree with linktitle and title, and weight

have avoided touching weight because there are alot of additions and i think we need to reweight a bunch of the pages

proposal - can we merge all 4 pr's into a seprate branch then do the reweighting in that branch then chuck it into main

Reply:
updated titles on all pages, - please review

Will address  page weights on separate PR